pydoctor/test/test_napoleon_docstring.py

```diff
@@ -466,6 +466,51 @@ class GoogleDocstringTest(BaseDocstringTest):
         """
 Single line summary
 
+Returns:
```

Comment:
I don't see where these new docstrings are tests.

And by a quick read I don't understand the testing principles for GoogleDocstringTest (no docstring for this class)

Instead of `Single line summary` my suggestion is to use a text describing the testing purpose of each of these examples.